packages/mongo/mongo_connection.js

```diff
@@ -792,9 +792,9 @@ MongoConnection.prototype.tail = function (cursorDescription, docCallback, timeo
   });
 
   return {
-    stop: function () {
+    stop: async function () {
       stopped = true;
-      cursor.close();
+      await cursor.close();
```

Comment:
This sounds like a potentially breaking change. Also, it becomes `void | Promise<void>`, since the `ObserveHandle.prototype.stop` does not return a `Promise`.

Reply:
I did a few tests, it doesnt affect the final result(close the cursors), so i removed the await